src/marqo/marqo_logging.py

```diff
@@ -6,6 +6,8 @@
 formatter = logging.Formatter(
     "{asctime} logger:'marqo' {levelname} {message}", style='{')
 ch = logging.StreamHandler()
-ch.setLevel(logging.INFO)
+
+# the ch stream handler won't accept any logs from a higher level than the one set here:
+ch.setLevel(logging.DEBUG)
```

Comment:
does this override the above line `mq_logger.setLevel(logging.INFO)`

Reply:
Basically they are two decoupled objects. It seems that the handler won't accept any logs at a higher level than its level. The logger won't accept any incoming logs higher than its log level. 

Before, because ch.Level=INFO, you could never output DEBUG statements to the console

Reply:
ok, that checks out!